projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/graphql/ExecutionDataFetcher.java

```diff
@@ -10,50 +10,32 @@
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionOrder.DIRECTION_FIELD;
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionOrder.PROPERTY_FIELD;
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionQueryVariables.FILTER_FIELD;
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionQueryVariables.ORDER_FIELD;
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionQueryVariables.PAGE_NUMBER_FIELD;
-import static com.vmware.taurus.service.graphql.model.DataJobExecutionQueryVariables.PAGE_SIZE_FIELD;
-
-import java.time.OffsetDateTime;
-import java.util.ArrayList;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Optional;
-import java.util.stream.Collectors;
-
+import com.vmware.taurus.datajobs.ToApiModelConverter;
+import com.vmware.taurus.datajobs.ToModelApiConverter;
+import com.vmware.taurus.service.JobExecutionFilterSpec;
+import com.vmware.taurus.service.JobExecutionRepository;
+import com.vmware.taurus.service.execution.JobExecutionService;
+import com.vmware.taurus.service.graphql.model.*;
+import com.vmware.taurus.service.graphql.strategy.datajob.JobFieldStrategyBy;
+import com.vmware.taurus.service.model.DataJobExecution;
+import com.vmware.taurus.service.model.ExecutionStatus;
 import graphql.GraphQLException;
 import graphql.schema.DataFetcher;
 import graphql.schema.DataFetchingEnvironment;
 import graphql.schema.SelectedField;
+import lombok.RequiredArgsConstructor;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.tuple.ImmutablePair;
-import org.springframework.data.domain.Page;
-import org.springframework.data.domain.PageImpl;
-import org.springframework.data.domain.PageRequest;
-import org.springframework.data.domain.Pageable;
-import org.springframework.data.domain.Sort;
+import org.springframework.data.domain.*;
 import org.springframework.data.jpa.domain.Specification;
 import org.springframework.stereotype.Component;
 
-import com.vmware.taurus.datajobs.ToApiModelConverter;
-import com.vmware.taurus.datajobs.ToModelApiConverter;
-import com.vmware.taurus.service.JobExecutionFilterSpec;
-import com.vmware.taurus.service.JobExecutionRepository;
-import com.vmware.taurus.service.graphql.model.DataJobExecutionFilter;
-import com.vmware.taurus.service.graphql.model.DataJobExecutionOrder;
-import com.vmware.taurus.service.graphql.model.DataJobExecutionQueryVariables;
-import com.vmware.taurus.service.graphql.model.ExecutionQueryVariables;
-import com.vmware.taurus.service.graphql.model.Filter;
-import com.vmware.taurus.service.graphql.model.DataJobPage;
-import com.vmware.taurus.service.graphql.model.V2DataJob;
-import com.vmware.taurus.service.graphql.strategy.datajob.JobFieldStrategyBy;
-import com.vmware.taurus.service.model.DataJobExecution;
+import java.time.OffsetDateTime;
+import java.util.*;
```

Comment:
avoid wildcard imports  - https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
I'm using the default import optimizer in IntelliJ ctrl-alt-O - is this not recommended?

Reply:
While I don't necessarily disagree with you I think this is opinion based and you can just as easily find opinions that favor using wildcard imports. And while our coding standard is still WIP we should think about putting stuff like this in there and be consistent with it since currently I can easily find examples of wildcard imports in our codebase.